drivers/storage/portworx/component/autopilot.go

```diff
@@ -171,6 +172,42 @@ func (c *autopilot) createConfigMap(
 			provider.Name, provider.Type, params)
 	}
 
+	if cluster.Spec.Autopilot.GitOps != nil {
+		config += "\ngitops:"
+
+		keys := make([]string, 0, len(cluster.Spec.Autopilot.GitOps.Params))
+		for k := range cluster.Spec.Autopilot.GitOps.Params {
+			keys = append(keys, k)
+		}
+		sort.Strings(keys)
+
+		params := ""
+		for _, key := range keys {
+			val := cluster.Spec.Autopilot.GitOps.Params[key]
+			if key == AutopilotDefaultReviewersKey {
+				switch val := val.(type) {
+				default:
+					return fmt.Errorf("invalid spec type for defaultReviewers: %T. Expected array of strings", val)
+				case []string:
+					params += fmt.Sprintf(`
+    %s:`, AutopilotDefaultReviewersKey)
+					for _, v := range val {
+						params += fmt.Sprintf(`
+      - "%s"`, v)
+					}
+				}
+				continue
+			}
+			params += fmt.Sprintf(`
+    %s: %s`, key, val.(string))
```

Comment:
Again, not a good idea to use `val.(string)`. This code will panic of the value is anything other than string.